pkgs/applications/audio/uhhyou.lv2/default.nix

```
@@ -0,0 +1,75 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, liblo, libGL, libX11, lv2, fftw }:
```

Personal preferences that make future diffs easier to review:

```suggestion
{ stdenv
, fetchgit
, pkgconfig
, libjack2
, liblo
, libGL
, libX11
, lv2
, fftw
}:
```